src/contexts/ServerContext.js

```diff
@@ -518,11 +521,24 @@ export const ServerProvider = ({ children }) => {
 
 
     const loadData = () => {
+        // not including servers key because we don't need to set default load amount here for it
+        const pageKeys = ["groupsPerPageKey", "resourcesPageKey", "usersPageKey"];
+
+        for (let key of pageKeys) {
+            if (!localStorage.getItem(environment[key])) {
+                localStorage.setItem(environment[key], environment.defaultAmount);
+            }
+        }
+
+        const rescPerPage = parseInt(localStorage.getItem(environment[pageKeys[0]]));
+        const usersPerPage = parseInt(localStorage.getItem(environment[pageKeys[1]]));
+        const groupsPerPage = parseInt(localStorage.getItem(environment[pageKeys[2]]));
```

Comment:
are these in the correct order?   you have [groups, resources, users] in alpha order on line 525.

Reply:
Ohhh, great catch. Was writing this code on a flight, must've slipped my eye 🙃